lib/shoryuken/queue.rb

```diff
@@ -5,7 +5,11 @@ class Queue
     def initialize(client, name)
       self.name   = name
       self.client = client
-      self.url    = client.get_queue_url(queue_name: name).queue_url
+      begin
+        self.url = client.get_queue_url(queue_name: name).queue_url
+      rescue Aws::SQS::Errors::NonExistentQueue
+        raise StandardError.new "Non existing queue: #{name}. Please create it in advance in SQS."
```

Comment:
This is wrong. This error is raised when you don't have permission to access the queue either. The default Aws exception is fine, I dont know why we have to raise a nondescript StandardError instead of a self explanatory `NonExistentQueue`. 

If we do end up adding this, we would also need specs for the logic change.